src/components/UniversalNavbar/UniversalNavbar.js

```diff
@@ -86,87 +83,74 @@ class UniversalNavbar extends React.Component {
           }}
         >
           <Layout.ScrollDetector>
-            <Media.PhoneOnly>
-              {/* Hamburger button */}
-              <div style={{ zIndex: 2, position: 'fixed', top: 24, right: 24 }}>
-                <TransformingBurgerButton
-                  showMobileMenu={showMobileMenu}
-                  clickHandler={this.toggleHamburger}
-                />
-              </div>
+            <div className={'UniversalNavbar--hamburger'}>
+              <TransformingBurgerButton
+                showMobileMenu={showMobileMenu}
+                clickHandler={this.toggleHamburger}
+              />
+            </div>
+
+            <div className={'UniversalNavbar--phoneOnly'}>
+              {/* Dark green mobile menu, shows up when hamburger is clicked */}
               <div
                 className={
-                  'universal-navbar-root' +
-                  ' ' +
-                  'universal-navbar-mobileRoot' +
-                  ' flex items-center justify-between'
+                  showMobileMenu ? 'shownMobileMenu' : 'hiddenMobileMenu'
                 }
               >
-                {/* Dark green mobile menu, shows up when hamburger is clicked*/}
-                <div
-                  className={
-                    showMobileMenu ? 'shownMobileMenu' : 'hiddenMobileMenu'
-                  }
-                >
-                  <a href="/">
-                    {LogoWhite({ className: 'universal-navbar-logo' })}
-                  </a>
-                  <Spacer.H56 />
-                  {navbarLinks.map((l) => (
-                    <div key={l.title + 'mobile'} style={{ marginBottom: 24 }}>
-                      <TitleXLarge.Sans.Regular400>
-                        <a key={l.title + 'mobile'} href={l.href}>
-                          {l.title}
-                        </a>
-                      </TitleXLarge.Sans.Regular400>
-                    </div>
-                  ))}
-                  <div style={{ position: 'absolute', bottom: 40 }}>
-                    <a href="/term" onClick={this.props.trackCtaClick}>
-                      <Button.Medium.BlackOutline>
-                        Check my price
-                      </Button.Medium.BlackOutline>
-                    </a>
+                <a href={LINKS.INDEX.href}>
+                  {LogoWhite({ className: 'universal-navbar-logo' })}
+                </a>
+                <Spacer.H56 />
+                {LINKS.NAVLINKS.map((l) => (
+                  <div key={l.title + 'mobile'} style={{ marginBottom: 24 }}>
+                    <TitleXLarge.Sans.Regular400>
+                      <a key={l.title + 'mobile'} href={l.href}>
+                        {l.title}
+                      </a>
+                    </TitleXLarge.Sans.Regular400>
                   </div>
+                ))}
+                <div style={{ position: 'absolute', bottom: 40 }}>
```

Comment:
Should we use a class instead of inline?

Reply:
Yes, I try to avoid inline styles. Similar to the previous comment though, I didn't want to refactor this entire file for a hotfix.

Reply:
(I.e. these styles were already here, and I just moved the wrappers and indentation.)

Reply:
I did move a few inline styles to Sass when they were more directly related to this ticket, i.e. the wrappers elements.